src/Microsoft.DotNet.Darc/DarcLib/LocalGitClient.cs

```diff
@@ -110,7 +110,7 @@ public async Task ResetWorkingTree(NativePath repoPath, UnixPath? relativePath =
         }
 
         // Also remove untracked files (in case files were removed in index)
-        result = await _processManager.ExecuteGit(repoPath, ["clean", "-df", relativePath], cancellationToken: CancellationToken.None);
+        result = await _processManager.ExecuteGit(repoPath, ["clean", "-xdf", relativePath], cancellationToken: CancellationToken.None);
```

Comment:
FYI `-xdff` would remove nested submodules that got deleted too, it's been my go to git clean command :)

Reply:
Ah, thanks. Fortunately, there's no submodules :)

Reply:
right now 😆

Reply:
But I see what you mean